modules/eventing/pages/eventing-advanced-keyspace-accessors.adoc

```diff
@@ -902,6 +935,10 @@ The positional parameter (denoted by _meta_ in the prototype above) represents t
 +
 The key of the document to be used for the operation. This is a mandatory parameter and must be of JavaScript _string_ type.
 
+** _meta.keyspace (type: Object)_
++
+The keyspace of the document to be used for the operation with the JavaScript format "keyspace":{"bucket_name":_string_,"scope_name":_string_,"collection_name":_string_}. This parameter is only required if you are using a a _binding_ with a wildcard for the scope and/or the collection. [.status]#Couchbase Server 7.1.1#
```

Comment:
```suggestion
The keyspace of the document to be used for the operation with the JavaScript format "keyspace":{"bucket_name":_string_,"scope_name":_string_,"collection_name":_string_}. 
This parameter is only required if you are using a _binding_ with a wildcard for the scope and/or the collection. [.status]#Couchbase Server 7.1.1#
```

Yes, all these repeated blocks have the same error, `a a _binding_,`, so it's definitely worth using an attribute or an included file for the common text if possible. (That way the error only needs correcting in one place).